appveyor.yml

```diff
@@ -72,10 +72,10 @@ install:
           else
             {$root = "C:\Miniconda-x64"}
           $env:path="$root;$root\Scripts;$root\Library\bin;$($env:path)"
+    - cmd: conda config --set show_channel_urls true
     - cmd: conda update --yes --quiet conda
     - cmd: set PYTHONUNBUFFERED=1
 
-    - cmd: conda config --set show_channel_urls yes
```

Comment:
> I was inspired to look based on [your change in `gdal`]( https://github.com/conda-forge/gdal-feedstock/pull/37 ).

I will removed that BTW. I noticed the channels were not listed and I had no idea if the dependencies were coming from `defaults` or `conda-forge`.